docs/pages/includes/database-access/db-introduction.mdx

```diff
@@ -0,0 +1,9 @@
+Access to {{ dbName }} can be provided by [Teleport Database
+Access](../../database-access/introduction.mdx). This allows for
+fine-grain access control through [Teleport's RBAC](../../database-access/rbac.mdx).
```

Comment:
```suggestion
Teleport can provide secure access to {{ dbName }} via the  [Teleport Database
Service](../../database-access/introduction.mdx). This allows for
fine-grained access control through [Teleport's RBAC](../../database-access/rbac.mdx).
```
Since we're moving away from positioning each protocol as a separate product.